onnxscript/rewriter/onnxruntime/instance_to_group_normalization.py

```diff
@@ -41,10 +41,10 @@ def check_if_simulated_instance_norm_is_used(
         bool: True if the simulated instance normalization is used, False otherwise.
     """
     weight_for_norm = _ir_utils.propagate_const_value(weight_for_norm)
-    weight_for_norm = _ir_utils.get_numpy_from_ir_value(weight_for_norm)
+    weight_for_norm = weight_for_norm.const_value
 
     bias_for_norm = _ir_utils.propagate_const_value(bias_for_norm)
-    bias_for_norm = _ir_utils.get_numpy_from_ir_value(bias_for_norm)
+    bias_for_norm = bias_for_norm.const_value
```

Comment:
I would call .numpy() if we need a numpy value; Besure to check for None before calling numpy; assign it to a new variable to prevent mypy complaining about the type changing for bias_for_norm;